scripts/lib-franklin.js

```diff
@@ -82,19 +82,41 @@ export function sampleRUM(checkpoint, data = {}) {
  * Loads a CSS file.
  * @param {string} href The path to the CSS file
  */
-export function loadCSS(href, callback) {
-  if (!document.querySelector(`head > link[href="${href}"]`)) {
-    const link = document.createElement('link');
-    link.setAttribute('rel', 'stylesheet');
-    link.setAttribute('href', href);
-    if (typeof callback === 'function') {
-      link.onload = (e) => callback(e.type);
-      link.onerror = (e) => callback(e.type);
+export async function loadCSS(href) {
+  return new Promise((resolve, reject) => {
+    if (!document.querySelector(`head > link[href="${href}"]`)) {
+      const link = document.createElement('link');
+      link.rel = 'stylesheet';
+      link.href = href;
+      link.onload = resolve;
+      link.onerror = reject;
+      document.head.append(link);
+    } else {
+      resolve();
     }
-    document.head.appendChild(link);
-  } else if (typeof callback === 'function') {
-    callback('noop');
-  }
+  });
+}
+
+/**
+ * Loads a non module JS file.
+ * @param {string} href The path to the CSS file
+ * @param {Object} attrs additional optional attributes
+ */
+
+export async function loadScript(url, attrs) {
```

Comment:
Why not having  a `callback` ? (and align the 2 methods)

Reply:
Also, `loadCSS` check for existing while `loadJS` does not... maybe we should align the 2 impl